client/apis/recipes.js

```diff
@@ -0,0 +1,21 @@
+import request from 'superagent'
+
+// const rootUrl = '/api/v1'
+
+// TODO: create .env file to store my KEY
+// Adjust config/setup -- Dotenv
+
+export function getRecipes(userInput) {
+  return request
+    .get(
+      // `https://api.edamam.com/api/recipes/v2?type=public&q=${userInput}&KEY`
+
+      `https://api.edamam.com/api/recipes/v2?type=public&q=${userInput}&app_id=eeb6d9e0&app_key=9bbd15d307d13bf463f6a7326338bced&nutrients%5BCA%5D=50%2B`
```

Comment:
This really shouldn't be here. Like you have said above put it in a env file. Especially before it gets merged into master. This is a must change.

Reply:
@ZinHoang is working on .ENV set up

Reply:
Completed, api key is now stored in .env file. Thank you.